app/src/main.py

```diff
@@ -122,7 +122,4 @@ async def main():
     await vehicle_app.run()
 
 
-LOOP = asyncio.get_event_loop()
-LOOP.add_signal_handler(signal.SIGTERM, LOOP.stop)
```

Comment:
You can remove the unused import on line 20

Reply:
Done!